modules/scale-up-autoscaler-hcp.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-machine-config.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="scale-up-autoscaler-hcp_{context}"]
+= Scaling up workloads in a hosted cluster
+
+On {hcp}, you can scale up workloads in your hosted cluster by using the cluster autoscaler.
+
+.Prerequisites
+
+* You have created the `HostedCluster` and `NodePool` resources.
+
+.Procedure
+
+. Enable cluster autoscaling for your hosted cluster to set the scaling behavior to `ScaleUpOnly` by running the following command:
++
+[source,terminal]
+----
+$ oc patch -n clusters hostedcluster lgao --type=merge --patch='{"spec": {"autoscaling": {"scaling": "ScaleUpOnly", "maxPodGracePeriod": 60}}}'
+----
+
+. Remove the `spec.replicas` field from the `NodePool` resource so that cluster autoscaler can manage the node count. Run the following command:
++
+[source,terminal]
+----
+$ oc patch -n clusters nodepool <node_pool_name> --type=json  --patch='[{"op": "remove", "path": "/spec/replicas"}]'
+----
+
+. Enable autoscaling to configure the minimum and maximum node counts for your node pools. Run the following command:
++
+[source,terminal]
+----
+$ oc patch -n <hosted_cluster_namespace> nodepool <nodepool_name>
+  --type=merge --patch='{"spec": {"autoScaling": {"max": 3, "min": 1}}}'
+----
+
+. Check that all worker nodes are in the `Ready` status by running the following command:
++
+[source,terminal]
+----
+$ oc --kubeconfig <hosted_cluster_name>.kubeconfig get nodes
+----
+
+. Ensure that the cluster autoscaler pod contains the `"--scale-down-enabled=false"` and `"--expander=priority,least-waste"` arguments by running the following command:
```

Comment:
I am not sure, but maybe we can ignore the part of the autoscaler pod arguments verification, it is internal used for debug purpose.